pkg/local_object_storage/engine/control.go

```diff
@@ -191,6 +191,14 @@ func (rCfg *ReConfiguration) AddShard(id string, opts []shard.Option) {
 
 // Reload reloads StorageEngine's configuration in runtime.
 func (e *StorageEngine) Reload(rcfg ReConfiguration) error {
+	e.mtx.Lock()
+	e.shardPoolSize = rcfg.shardPoolSize
```

Comment:
Not sure it needs to be updated at all, nothing really uses it.

Reply:
Below in the [function](https://github.com/nspcc-dev/neofs-node/blob/50e3ff4ec6546823e430df924d27e752380e3484/pkg/local_object_storage/engine/control.go#L268), when adding shards, it is used.

https://github.com/nspcc-dev/neofs-node/blob/f1b69823ac8a97b70f4f1981ebb82b30a99511c5/pkg/local_object_storage/engine/shards.go#L113-L117

Reply:
That's init mostly, but we can also manage shards at runtime, so probably yes, it can be used.

Reply:
> but we can also manage shards at runtime

it is supported